openbb_platform/providers/biztoc/openbb_biztoc/utils/helpers.py

```diff
@@ -78,7 +78,7 @@ def get_all_tags(api_key) -> Dict[str, List[str]]:
 
 def get_news(
     api_key: str,
-    filter: Literal[
+    filter_: Literal[
```

Comment:
this remembered me that we have a lot of python keywords laying around in the models and we should probably to this for those cases (like `type` and `open`)